orchestration/dagster_orchestration/hca_orchestration/solids/load_hca/load_table.py

```diff
@@ -9,13 +9,13 @@
 from hca_orchestration.resources.config.hca_dataset import TargetHcaDataset
 from hca_orchestration.resources.config.scratch import ScratchConfig
 from hca_orchestration.solids.data_repo import wait_for_job_completion
-from hca_orchestration.solids.load_hca.data_files.typing import FileMetadataType, FileMetadataTypeFanoutResult
 from hca_orchestration.solids.load_hca.poll_ingest_job import check_data_ingest_job_result
-from hca_orchestration.support.typing import HcaScratchDatasetName
+from hca_orchestration.support.typing import HcaScratchDatasetName, MetadataType, MetadataTypeFanoutResult
 
 
 def _diff_hca_table(
-        file_metadata_type: FileMetadataType,
+        metadata_type: MetadataType,
+        is_file_metadata: bool,
```

Comment:
truly only adding this here because we need to be able to point to a different source_path based on whether it is file metadata or non-file metadata